bin/injector/Dockerfile

```diff
@@ -30,7 +30,6 @@ COPY --from=binaries /usr/lib/x86_64-linux-gn[u] /usr/lib/x86_64-linux-gnu/
 COPY --from=binaries /usr/bin/test /bin/sh
 
 ARG TARGETARCH
-ENV BPF_DISK_FAILURE_NAME "bpf-disk-failure-${TARGETARCH}"
```

Comment:
seems unrelated?

Reply:
Indeed, this environment variable is never used by the diskfailure.